### DISCUSSION TITLE
Do we have anything like mkdocs theme?

### DISCUSSION THREAD

**Discussion by ghost** [beercss/beercss#134] [Q&A]
I'm writing a document using mkdocs-material but I like md3 

So I'm wondering if we have a similar project like that
~~If not I'm going to make one~~

**Comment by KicKerBNU** [comment]
Hello @lotusproxy we dont have a project like that and sounds good to me to have one. 
Today our updates goes to:
website:  [https://www.beercss.com/](https://www.beercss.com/)
docs: [https://github.com/beercss/beercss/tree/main/docs](https://github.com/beercss/beercss/tree/main/docs)
releases: [https://github.com/beercss/beercss/releases](https://github.com/beercss/beercss/releases)
codepen: [https://codepen.io/collection/XydYMB](https://codepen.io/collection/XydYMB)
